src/django_peertube_runner_connector/utils/ffprobe.py

```diff
@@ -28,13 +28,17 @@ def get_video_stream_dimensions_info(path: str, existing_probe=None):
     """Return the video stream dimensions info."""
 
     if video_stream := get_video_stream(path=path, existing_probe=existing_probe):
+        width = video_stream.get("width", 0)
+        height = video_stream.get("height", 0)
+        min_dimension = min(height, width)
+        max_dimension = max(height, width)
+
         return {
-            "width": video_stream["width"],
-            "height": video_stream["height"],
-            "ratio": max(video_stream["height"], video_stream["width"])
-            / min(video_stream["height"], video_stream["width"]),
-            "resolution": min(video_stream["height"], video_stream["width"]),
-            "isPortraitMode": video_stream["height"] > video_stream["width"],
+            "width": width,
+            "height": height,
+            "ratio": max_dimension / min_dimension if min_dimension > 0 else 0,
+            "resolution": min_dimension,
+            "isPortraitMode": height > width,
```

Comment:
How does the whole process behaves if we get `0` in all values ?

Reply:
If we get 0 in values the process will continue and transcode with 0x0 dimensions. This is not what we want so edited the PR so core properties (height, width, filename, duration) will crash if are not in the video